src/fparser/common/readfortran.py

```diff
@@ -1047,6 +1077,28 @@ def warning(self, message, item=None):
 
     # Auxiliary methods for processing raw source lines:
 
+    def handle_cpp_directive(self, line):
+        """Determine whether the current line is likely to hold
+        C preprocessor directive.
+
+        If the first character of a line is the symbol ``#`` it is assumed
+        this is a preprocessor directive. This means any whitespace in
+        front of the directive are not recognized.
+
+        Preprocessor directives can be used only in Fortran codes.  They are
+        ignored when used inside PYF files.
+
+        The actual line content is not altered.
+
+        :param str line: the line to be tested for directives.
+        :return: A tuple containing the line and True/False depending on \
```

Comment:
Could you please add a blank line above this line to separate the input arguments from the return values.